requirements.txt

```diff
@@ -3,3 +3,4 @@ filelock
 numpy
 boto3[crt]
 requests
+google-cloud-storage
```

Comment:
Can we move this to extras? This shouldn't be a default

Reply:
sounds good, i figured boto3 was here as well so i added it here

Reply:
Yes, we will make it first class citizen when there is more demand for it